### DISCUSSION TITLE
TOTUM PRO, VPS и CMS

### DISCUSSION THREAD

**Discussion by horyachka** [totumonline/totum-ru-issues-and-discussions#239] [Как это сделать?] [answered]
Добрый день!
Для публикации всего проекта нужно залить на VPS две части проекта:

Часть 1. TOTUM PRO
1. VPS сервер с установленным Ubuntu 20.04 заказал для тестирования (параметры для старта, чтобы заполнить проект: 1 ядро; 1024 МБ; 15 ГБ SSD)

Часть 2. CMS
1. Нужно залить сайт, созданный на CMS Mobirise – это оффлайн конструктор сайтов для Windows, Mac, Linux.
2. Сайт, как визитка, информирует посетителей о проекте и домен закреплен за сайтом, а поддомен - ссылка на TOTUM.

Вопрос:
1. Можно ли на указанный VPS установить TOTUM PRO и CMS вместе?
2. Или на VPS установить только TOTUM PRO, а CMS устанавливать в другом месте?

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Вам нужен будет нормальный скилл в linux, что бы сконфигурировать еще один сайт для nginx. Потребуется изменить конфиг nginx тотума, так-как по умолчанию он предполагает, что Totum один на web-сервере.

Далее пути указаны для результата работы скрипта автоустановки. Если вы делали вручную, то используйте заданные вами пути:

В  `/etc/nginx/sites-available/totum.conf`:

`server_name _;` —> `server_name totum_domain.domain.zone;`

Дальше типа по этой инструкции без установки nginx (тк он установлен к этому моменту): https://ru.docs.totum.online/ubuntu#nginx

Для сайта какой-то такой должен быть:

```
server {
    listen *:80;
    server_name DOMAIN.ZONE;
    charset UTF-8;
    root /home/totum/SITE_FOLDER_HERE;
    index index.php;
    expires off;

    client_max_body_size 10M;

    location / {
    try_files $uri $uri/ /index.php?$args;
    }

    location ~* \.php$ {

    fastcgi_param PHP_ADMIN_VALUE "sendmail_path = /usr/sbin/sendmail -t -i -f admin@youdomain.input";
    fastcgi_pass unix:/run/php/php8.0-fpm-totum.sock;
    fastcgi_split_path_info ^((?U).+\.ph(?:p\d*|tml))(/?.+)$;
    try_files $uri =404;
    include fastcgi_params;
    fastcgi_param  SCRIPT_FILENAME  $document_root$fastcgi_script_name;
    fastcgi_read_timeout 300;
 }
}
```

Применяется все при перезапуске nginx.

Дальше надо получить сертификат, как-то так: https://ru.docs.totum.online/ubuntu#certbot начиная с шага «Подключим эту дополнительную конфигурацию...»

Шаг «проверить список доменов в сертификате» будет работать при указании папки домена вместо `*`.

Как-то так и может я чего забыл.

**Reply by horyachka** [reply]
Алексей, спасибо!
Следующие шаги понятны. Вопрос закрыт.
